java/client/src/main/java/glide/api/commands/ConnectionCommands.java

```diff
@@ -0,0 +1,29 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Connection Management Commands interface.
+ *
+ * @see: <a href="https://redis.io/commands/?group=connection">Connection Management Commands</a>
+ */
+public interface ConnectionCommands {
+
+    /**
+     * Ping the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
+     * @return Response from Redis containing a <code>String</code>.
```

Comment:
same - please follow the functions' documentation we have in python/node throughout the PR

Reply:
Updated all commands in 8388c49de3819c0be4a4b3b9f5f3ffa2b21e81ad